kotsadm/web/src/components/troubleshoot/GenerateSupportBundle.jsx

```diff
@@ -275,11 +284,4 @@ class GenerateSupportBundle extends React.Component {
   }
 }
 
-export default withRouter(compose(
-  withApollo,
```

Comment:
is this imported on top? (withApollo,compose,graphql) we should delete it because it's unused

Reply:
yep, good catch